test/jdk/java/lang/Object/FinalizationOption.java

```diff
@@ -89,7 +89,7 @@ static boolean checkFinalizerCalled(boolean expected) {
         for (int i = 0; i < 100; i++) {
             System.gc();
             try {
-                Thread.sleep(10L);
+                Thread.sleep(30L);
```

Comment:
I think it would be better to scale these kinds of sleep time bounds by the timeout factor used to run the jtreg test, which should already to scaled up for known-to-be-slower testing scenarios.

There is a utility method to return this value in jdk/test/lib/Utils.java

Reply:
I guess that sort of makes sense.  The default timeout factor is 4 I think so I guess I descale the existing total wait of 1000 ms so that the total wait is now Utils.adjustTimeout(250)?  Maybe others might offer an opinion?  Since these waits are a fixed timeout with no early exit I don't think we want them to be too long but anything in 1-5 seconds really doesn't matter.  That's part of why I was just picking a static value.

Reply:
> I guess that sort of makes sense. The default timeout factor is 4 I think so I guess I descale the existing total wait of 1000 ms so that the total wait is now Utils.adjustTimeout(250)? Maybe others might offer an opinion? Since these waits are a fixed timeout with no early exit I don't think we want them to be too long but anything in 1-5 seconds really doesn't matter. That's part of why I was just picking a static value.

Looking at running the test suite as a whole, especially for tier 1 tests which are run very frequently, a few seconds here, a few seconds there, over a few thousand tests starts to add up in terms of increasing the minimum time needed to run the sets of tests.

(Separate from this issue, I think it would be a useful exercise to more systematically examine the constant sleeps in the test suite and tune down the constant factor to let the timeout factor do more of the heavy lifting on avoiding spurious failures.)